### PR TITLE
Cellsystems dataset

### PR DESCRIPTION
**Pull request by meganrm**

### CODE REVIEW

File: dataset-cellsystems-2021/fish/gene_pairs.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        "GenePairName": "BMPER-VCAN",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "ACTN2-TTN",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "TTN-ACTN2",
+        "Channels": "561-638"
```
Comment (meganrm) [inline dataset-cellsystems-2021/fish/gene_pairs.json:40]: @tanyasg  in the long run we will have a better way of dealing with this, but for now, these should look like this: 
```    "StructureId/Name": "TTN-ACTN2",
        "ProteinId/DisplayName": "TTN-ACTN2",
        "CellLineId/Name": "TTN-ACTN2",
        "GeneId/Name": "TTN-ACTN2",
        "GeneId/FullName": "TTN-ACTN2",
        "ProteinId/Name": "TTN-ACTN2",
        "Channels": "561-638"
```

Comment (meganrm) [reply dataset-cellsystems-2021/fish/gene_pairs.json:40]: And "TTN-ACTN2" should be in the `file_info` where cell line normally goes in cell_feature_analysis json

File: dataset-cellsystems-2021/live/feature_defs.json [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    "key": "cell-aspect",
+    "discrete": false
+  },
+  {
+    "displayName": "Experiment",
+    "description": "Differentiation experiment encompassing all plates and wells setup and replated on a given day.",
+    "tooltip": "Differentiation experiment encompassing all plates and wells setup and replated on a given day.",
+    "unit": "",
+    "key": "diff-exp",
+    "discrete": true
```
Comment (meganrm) [inline dataset-cellsystems-2021/live/feature_defs.json:142]: if a feature is discrete it needs to have the `options` list

Comment (tanyasg) [reply dataset-cellsystems-2021/live/feature_defs.json:142]: can the options list just have a name?
 ```
   "options": {
        "1": {
            "name": "Exp1"
        },
        "2": {
            "name": "Exp2"
        },
        "3": {
            "name": "Exp3"
        }
```

Comment (meganrm) [reply dataset-cellsystems-2021/live/feature_defs.json]: I think currently it needs a color. If you're having trouble picking them, this is a good site: https://colorbrewer2.org/#type=sequential&scheme=BuGn&n=3

**Pull request closed without merging**